### PR TITLE
Migrate tracking modules to scilpy 2.0.0 + nf-test.

### PR DESCRIPTION
**Pull request by arnaudbore**
## Type of improvement

**If submitting a new module or fixing a bug, please use the appropriate template.**

- [ ] Documentation
- [ ] Development tools (e.g. linter, formatter, etc.)
- [ ] Development container
- [x] Global update (please specify)
- [ ] Other (please specify)

## Describe your improvement

Migrate trackings modules to scilpy 2.0.0 + nf-test.

## Checklist before requesting a review

- [x] Ensure the syntax is correct (**EditorConfig** and **Prettier** must pass)
- [x] Run the test suites if your changes affect any module
- [ ] Regenerate the **Poetry** lock file if you have updated the dependencies
- [ ] Ensure the documentation is up-to-date

#79

### CODE REVIEW

File: modules/nf-scil/tracking/localtracking/environment.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name: "tracking_localtracking"
+channels:
+  - Docker
+  - Apptainer
+dependencies:
+  - "scilpy"
```
Comment (gagnonanthony) [inline modules/nf-scil/tracking/localtracking/environment.yml:6]: Minor comment, but I think everywhere else, `scilpy` is written with a capital S.

File: modules/nf-scil/tracking/localtracking/tests/main.nf.test [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+                process {
+                    """
+                    input[0] = Channel.from( [ "tracking.zip" ] )
+                    input[1] = "test.load-test-data"
+                    """
+                }
+            }
+        }
+
+    test("stracking - localtracking - fa") {
```
Comment (gagnonanthony) [inline modules/nf-scil/tracking/localtracking/tests/main.nf.test:27]: Is `stracking` voluntary, or is it a typo?

**Review by gagnonanthony — CHANGES_REQUESTED** [review]
Great job! 💯  I wrote minor comments (mostly typos), but I wondered if you forgot to migrate the `localtracking` module to `scilpy 2.0`? I can't seem to be able to see the changes, but it might be only me!

File: modules/nf-scil/tracking/localtracking/main.nf [hunk 3]
```diff
@@ -115,7 +115,11 @@ process TRACKING_LOCALTRACKING {
 
     cat <<-END_VERSIONS > versions.yml
     "${task.process}":
-        scilpy: 1.6.0
+<<<<<<< HEAD
```
Comment (AlexVCaron) [inline modules/nf-scil/tracking/localtracking/main.nf:118]: Merge error

**Review by AlexVCaron — CHANGES_REQUESTED** [review]

**Pull request merged**